src/ballet/toml/fd_toml.h

```diff
@@ -43,8 +44,12 @@ FD_PROTOTYPES_BEGIN
      inline table  | x={a=1,b=2} | subpod
      inline array  | x=[1,2]     | subpod (keys %d formatted cstrs)
      bool          | true        | int
-     integer       | -3          | ulong (two's complement)
-     float         | 3e-3        | double
+     dec int       | -3          | long
+     hex int       | 0x3         | ulong
+     oct int       | 0o3         | ulong
+     bin int       | 0b11        | ulong
+     float         | 3e-3        | float
+     float special | "+inf"      | cstr ("-inf", "+inf", "nan")
```

Comment:
I also don't see why this needs to exist? Just map +inf to the +Infinity float encoding?

Reply:
See my other comment.